js/background.js

```diff
@@ -183,8 +183,8 @@ const filter = {
     types: ["main_frame", "sub_frame", "stylesheet", "script", "image", "object", "xmlhttprequest", "other"]
 };
 browser.webRequest.onBeforeRequest.addListener(requestUpdated, filter, ['blocking', 'requestBody']);
-browser.webRequest.onBeforeSendHeaders.addListener(requestUpdated, filter, ['requestHeaders', 'extraHeaders']);
-browser.webRequest.onSendHeaders.addListener(requestUpdated, filter, ['requestHeaders', 'extraHeaders']);
```

Comment:
Hmm, but won't this make Chrome stop sending us the `Referer` header? I think I had to add `extraHeaders` in there for Chrome at one point so we could get that.

Reply:
We might have to have some branching in this code (asking for `extraHeaders` in Chrome but not Firefox) or a separate manifest (to ask for more permissions in Firefox but not in Chrome). I'm okay if either of those if that's what it takes. But getting the `Referer` header is kind of important to the recordings since it allows us to connect events together and tell which is a parent of which.

Reply:
@azhawkes Right.. ok, I put it back for Chrome. Not sure of Firefox solution yet. Do we modify referrer-policy? Is it `Origin` or something else?